tests/unit/text.t

```diff
@@ -61,4 +62,25 @@ is(normalize_percentages('2500%', 'fr'), "2\N{U+00A0}500\N{U+00A0}%");
 is(normalize_percentages('2,50%', 'en'), "2.5%");
 is(normalize_percentages('2.50%', 'en'), "2.5%");
 
+# Test preprocess_product_field
```

Comment:
in text.t, we should make a unit test only for the remove_email() function of Text.pm, you can test it with a few different values that have (or don't have) email addresses

Reply:
Okok...doing rn